semgrep/rules/python/xss/reflected_xss.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+  - id: reflected-cross-site-scripting
+    patterns:
+      - pattern-not-inside: |
+          import phantom
+          ...
+          class $CLASS(phantom.BaseConnector):
```

Comment:
are we ignoring classes that import base connector because it automatically overrides the content type or?

Reply:
I added this clause since I couldn't find any rest handler implementations being inside of a connector class, and thought it would be useful to prevent false positives

but it seems like connector methods would never return `HttpResponse` so maybe this clause isn't needed?

Reply:
what would be a false positive in this scenario? is there ever a case we would need to return a non-text HttpResponse? the app is presumably always manually creating the HttpResponse for the platform to use with some arbitrary values since since it's a django object iinm. the app itself wouldn't really have a need to use a django object for its own internal use

Reply:
right, I was considering the possibility of an app using `HttpResponse` in some internal method, but this doesn't seem likely on second thought, and action handlers will always return a response using the `BaseConnector` API

will remove the clause